src/shapes/group.class.js

```diff
@@ -83,25 +96,32 @@
 
     /**
      * @private
+     * @param {Boolean} [unchangeCoords] if true, coordinates of objects enclosed in a group do not change
      */
-    _updateObjectsCoords: function() {
-      this.forEachObject(this._updateObjectCoords, this);
+    _updateObjectsCoords: function(unchangeCoords) {
+      for (var i = this._objects.length; i--; ){
+        this._updateObjectCoords(this._objects[i], unchangeCoords);
+      }
     },
 
     /**
      * @private
-     */
-    _updateObjectCoords: function(object) {
-      var objectLeft = object.getLeft(),
-          objectTop = object.getTop(),
-          center = this.getCenterPoint();
-
-      object.set({
-        originalLeft: objectLeft,
-        originalTop: objectTop,
-        left: objectLeft - center.x,
-        top: objectTop - center.y
-      });
+     * @param {Object} object
+     * @param {Boolean} [unchangeCoords] if true, coordinates of object dose not change
+     */
+    _updateObjectCoords: function(object, unchangeCoords) {
+      if (!unchangeCoords) {
+        var objectLeft = object.getLeft(),
+            objectTop = object.getTop(),
+            center = this.getCenterPoint();
+
+        object.set({
+          originalLeft: objectLeft,
```

Comment:
No need for aliasing here as it's so simple. Let's just use `object.getLeft()`, `object.getTop()` inline